examples/commands.reference.md

```diff
@@ -4,6 +4,7 @@
 |------------------------------------------	|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------	|--------------------------------	|----------------------------------	|-------------------------------	|-------------------------------------------------------------------------------------------	|-------------------------------------------------------------------------------------------------------------------------------------------------	|
 | vscode.didact.cliCommandSuccessful 	| Even simpler command-line check for system capabilities. Takes two parameters - the id of the HTML element to update with the results and the test command (such as mvn --version. If command executes with no errors (return code 0), it returns true. If not, false. 	| requirement: string / required 	| testCommand: string / required 	|  	| true on success, false on failure; result posted as message to update requirement element 	| https://github.com/redhat-developer/vscode-didact/wiki/Available-Commands-in-Didact-and-Elsewhere#vscodedidactclicommandsuccessful 	|
 | vscode.didact.closeNamedTerminal 	| Terminate a named terminal and dispose of it. 	| name : string / required 	|  	|  	| Check Didact output channel 	| https://github.com/redhat-developer/vscode-didact/wiki/Available-Commands-in-Didact-and-Elsewhere#vscodedidactclosenamedterminal 	|
+| vscode.didact.copyFileTextToClipboardCommand  | Copies text from a file to the system clipboard. | text : string / required |  	|  	| Check Didact output channel 	| https://github.com/redhat-developer/vscode-didact/wiki/
```

Comment:
the problem of this approach is that it won't work in Che by default for tutorials written in other extensions than Didact.

Reply:
That is not a problem with this PR, it's a problem with how Che silos work isolating each individual plug-in from the others. Not going to be solved here.